tests/src/integration/masp.rs

```diff
@@ -1248,3 +1245,90 @@ fn wrapper_fee_unshielding() -> Result<()> {
     assert!(tx_run);
     Ok(())
 }
+
+// Test that a masp unshield transaction can be succesfully executed even across
+// an epoch boundary.
+#[test]
+fn cross_epoch_tx() -> Result<()> {
```

Comment:
Can I assume that assume that CI integration test failure is due solely to the additional time taken by this test?